services/channel/service/migrations/current.sql

```diff
@@ -0,0 +1,890 @@
+--! Message: added initial tables
```

Comment:
These contents are duplicating the initial tables migration and should be dropped